client/dashboard/sites/actions.tsx

```diff
@@ -52,6 +52,24 @@ export function getActions( router: AnyRouter ): Action< Site >[] {
 			},
 			isEligible: ( item: Site ) => canManageSite( item ),
 		},
+		{
+			id: 'restore',
+			isPrimary: true,
+			icon: backup,
+			label: __( 'Restore site' ),
+			isEligible: ( item: Site ) =>
+				!! item.is_deleted && ! isP2( item ) && ! isSelfHostedJetpackConnected( item ),
```

Comment:
`!! item.is_deleted` can be just `item.is_deleted` because `is_deleted` is a boolean already, not `?: boolean`.